tools/offline_log_viewer/storage.py

```diff
@@ -133,6 +133,7 @@ class BatchType(Enum):
     archival_metadata = 19
     cluster_config_cmd = 20
     feature_update = 21
+    tx_checkpoint = 22
```

Comment:
We should have two new batches e.g. `send_tx_leadership` & `receive_tx_leadership`. When we reuse the same batch we overload its meaning depending on the context. It's harder to describe what it does and it may lead to situation when we invoke one code path instead of another. With complex code it's safer to be verbose rather than smart.